mipengine/udfgen/py_udfgenerator.py

```diff
@@ -349,6 +349,7 @@ def __init__(
         self,
         udfregistry: UdfRegistry,
         func_name: str,
+        *,
```

Comment:
What is this?

Reply:
This is a python language feature. See [PEP 3102 – Keyword-Only Arguments](https://peps.python.org/pep-3102/). 

Since arguments of `PyUdfGenerator` and `AdhocUdfGenerator` are slightly different, I force them to be keyword-only to avoid potential confusion.